src/redux/copyConfigurationExtended/copyConfigurationExtendedSlice.js

```diff
@@ -206,7 +205,7 @@ export const copyConfigurationExtendedSlice = createSlice({
 
 export const getConfigurations = createAsyncThunk(GET_CONFIGURATIONS_ACTION, async (_, { getState, rejectWithValue }) => {
   const state = getState()
-  const credentials = { userKey: state.credentials.credentials.userKey, secret: state.credentials.credentials.secretKey }
+  const credentials = { userKey: state.credentials.credentials.userKey, secret: state.credentials.credentials.secretKey, gigyaConsole: window.location.hostname }
```

Comment:
Instead of using `window.location.hostname` everywhere, we can apply the same strategy used for `window.location.hash` with `state.copyConfigurationExtended.currentSiteApiKey`.

Also, it would be interesting to consider storing the `gigyaConsole` in the `credentialsSlice`, instead of this slice, as they are mostly used together.